### PR TITLE
Rename build_win to check_win_* and rename rhub_check to check_rhub

### LINKED ISSUE #1598: check function naming
Should we rename `build_win()` to `check_win()` and `rhub_check()` to `check_rhub()` to make the names more consistent and easier to discover?

### PR DESCRIPTION
**Pull request by jimhester**
Fixes #1598

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # devtools 1.13.3.9000
 
+* `build_win()` has been renamed to `check_win_release()`, `check_win_devel()`,
+  check_win_oldrelease()` (#1598).
```
Comment (hadley) [inline NEWS.md]: Missing `` ` ``

File: R/check-win.r [hunk 2]
```diff
@@ -10,21 +10,34 @@
...
 #' @export
 #' @family build functions
-build_win <- function(pkg = ".", version = c("R-release", "R-devel"),
+#' @name check_win
+NULL
+
+#' @describeIn check_win Check package on the development version of R.
+#' @export
+check_win_devel <- function(pkg = ".", args = NULL, quiet = FALSE) {
+  check_win(pkg = pkg, version = "R-devel", args = args, )
```
Comment (hadley) [inline R/check-win.r]: Extra `,`?

**Comment by jimhester** [comment]
Eeek 🙈 , sorry, the perils of not actually running the code. Both issues now fixed.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**